SimplenoteTests/RemoteResult+TestHelpers.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+@testable import Simplenote
+
+extension Remote {
+    static func randomResult() -> Result<Data?, RemoteError> {
+        let random = arc4random_uniform(1)
```

Comment:
Out of curiosity!!! I've noticed the original code uses `Bool.random()`. What'd be the reason for this change? (purely curious!!)

Reply:
Previously the result of the closures were capturing a Bool and an Error, and this function.

Since I was changing it to no longer returning bool I went to do a random in a different way.  But since you asked, I realized that it was still a binary choice and I could use Bool.Random to choose success or failure, and then return the correct value. Have updated the function